tests/e2e/entities/test_contract_deploy.py

```diff
@@ -270,21 +337,54 @@ def filtered_contract_query(_filter):
                     }
                 }
             }
-        })
+        },
+            'CONTRACTS_BY_STORE_CONTRACT_MESSAGES_CODE_ID_ASC'
+        )
 
         # query contract, filter by contract address
         filter_by_id_equals = filtered_contract_query({
             "id": {
-                "equalTo": str(self._contract.address)
+                "equalTo": str(self.entities['contractEntity']['equal'][ContractFields.id.value])
             }
-        })
+        },
+            'CONTRACTS_BY_STORE_CONTRACT_MESSAGES_CODE_ID_ASC'
+        )
 
         # query contract, filter by interfaces
         filter_by_interfaces_equals = filtered_contract_query({
             "interfaces": {
                 "isNull": False
             }
-        })
+        },
+            'CONTRACTS_BY_STORE_CONTRACT_MESSAGES_CODE_ID_ASC'
+        )
+
+        store_contract_nodes = "{codeId}"
+        instantiate_contract_nodes = "{codeId}"
+
+        def filtered_store_contract_message_query(_filter, order=""):
+            return test_filtered_query("storeContractMessages", _filter, store_contract_nodes, _order=order)
+
+        def filtered_instantiate_contract_message_query(_filter, order=""):
+            return test_filtered_query("instantiateContractMessages", _filter, instantiate_contract_nodes, _order=order)
```

Comment:
Let's double check whether or not we need this or if we can re-use a common pair of these functions.